tests/redisearch.ts

```diff
@@ -28,8 +28,19 @@ describe('RediSearch Module testing', async function() {
         });
         await client.connect();
         await redis.connect();
+        try {
+            await client.redis.flushall();
+            await client.dropindex(index);
+            await client.dropindex(`${index}1`);    
+            await client.dropindex(`${index}-searchtest`);    
+            await client.dropindex(`${index}-droptest`);    
+        } catch (error) {
+            // console.error(error);
+        }
     })
     after(async () => {
+        await client.dropindex(index);
+        await client.dropindex(`${index}1`);
```

Comment:
Why are you dropping indexes?

Reply:
for running clean test on every execution